README.md

```diff
@@ -252,20 +252,24 @@ The following generic methods are added to `nemo.view`
 
 `@returns {Promise}` resolves to true or rejected
 
-#### _wait(locatorString[, timeout])
+#### _wait(locatorString[, timeout, [msg]])
```

Comment:
sorry to be trivial yet again, but please put the comma inside the inner [ ]. I.e.
`_wait(locatorString[, timeout[, msg]])`